internal/overlord/servstate/handlers.go

```diff
@@ -373,10 +374,14 @@ func (s *serviceData) startInternal() error {
 		// started (previous logs have already been copied).
 		outputIterator = s.logs.HeadIterator(0)
 	}
+
+	// set up output logging for the command
 	serviceName := s.config.Name
 	logWriter := servicelog.NewFormatWriter(s.logs, serviceName)
-	s.cmd.Stdout = logWriter
-	s.cmd.Stderr = logWriter
+	forwarder := logstate.NewLogForwarder(s.manager.logMgr.GetTargets, serviceName)
```

Comment:
so we need to remove this log forwarder and the io.MultiWriter here, we should be using s.logs